services/blockchain-indexer/methods/dataService/controllers/blockchainApps.js

```diff
@@ -40,14 +40,25 @@ const getBlockchainApps = async (params) => {
 	// }
 
 	const result = {
-		data: [{
-			name: 'Test app',
-			chainID: 'aq02qkbb35u4jdq8szo3pnsq',
-			state: 'active',
-			address: 'lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu',
-			lastCertificateHeight: 1000,
-			lastUpdated: 123456789,
-		}],
+		data: [
+			{
+				name: 'Lisk',
+				chainID: 100,
```

Comment:
chainID for `Lisk` is `1`.
```suggestion
				chainID: 1,
```